### DISCUSSION TITLE
Welcome to netdata-ios Discussions!

### DISCUSSION THREAD

**Discussion by arjunkomath** [arjunkomath/netdata-ios#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by celogeek** [comment]
Hi, is it possible to add graphs or pie charts to the metrics we want to follow on the main screen ?

**Reply by arjunkomath** [reply]
Hey, it's not possible to add custom charts at the moment. This feature is in pipeline.

**Reply by celogeek** [reply]
nice. I wait for it. thanks a lot for this app !

**Comment by i7voodoo** [comment]
Hello,
I purchased the app and I'm a bit confused. Would you give me an example URL that should be put into the app?
Although it is mentioned that the app does not work with netdata cloud, the installation guide leads to netdata.cloud!

**Reply by arjunkomath** [reply]
Hey,

Here are some example URL for Netdata server:
https://london3.my-netdata.io
https://cdn77.my-netdata.io

Yes, you're correct, the installation documentation is hosted in netdata.cloud but its not same as Netdata.
Netdata is an open source monitoring tool -> https://github.com/netdata/netdata
Netdata cloud is a SaaS product which offers the open source monitoring tool as a service -> https://www.netdata.cloud
I hope that clears the confusion.

**Reply by i7voodoo** [reply]
Thank you for your response.
So I should remove the current installation? and then install the one in Github? I need a clear path to follow.
Another question: would my dashboard be publicly available?
Thank you.

**Reply by arjunkomath** [reply]
You can follow these steps:

- Uninstall current setup
- Install the open source version of Netdata agent
- Confirm that you’re able to access the netdata web UI using your host name or IP address at port 19999, for example 192.168.0.100:19999
- You can try accessing the url from mobile to make sure your mobile device also has access to the server
- Finally add the server to netdata app

I cannot answer whether your dashboard will be public or not as I have no information about your infrastructure.
What I can confirm is that the netdata web UI is just like any other web app, it doesn’t have anything authentication, it will be available to the public if your machine is exposed to the internet and port 19999 is open.

Potential options for restricting access:
- Use a reverse proxy like Nginx with basic authentication
- Use a private VPN for accessing your server from your personal devices, like https://tailscale.com or https://www.zerotier.com

Personally I prefer the private VPN as it’s more secure and offers fine grain control over the network.
Hope that answers all your question.

**Reply by matzel82** [reply]
That mean I can’t use the app mit Netdata.cloud ???

**Reply by arjunkomath** [reply]
> That mean I can’t use the app mit Netdata.cloud ???

Yes, the app does NOT work with Netdata cloud. The app works by connecting directly to the server.
